docker/dev.yml

```diff
@@ -0,0 +1,19 @@
+version: '3'
+
+services:
+  facets-api:
+    ports:
+      - "${API_EXPOSE}:80"
+    container_name: knowledge-panel
```

Comment:
We don't add a static container_name, because it would disable running more than one project on same machine, we let docker-compose create container name.